addons/website_slides/static/src/scss/website_slides.scss

```diff
@@ -561,6 +561,13 @@ $line-height-truncate: 1.25em;
             }
         }
 
+        .o_wslides_lesson_aside_list_links i {
+            padding-right: 10px;
```

Comment:
Can't it be solved with pe class ?

Reply:
No sadly it's either too wide or too narrow :confused: It's between pe-2 and pe-3.
Same than what I explained above: Here the done button is using css classes so it's great but the links icons have a different width than the done button so to align their name with the slide title we'll need the exact number of pixels :confused: